index.html

```diff
@@ -549,13 +534,7 @@ <h3>
           <li>If |json|["lang"] doesn't [=map/exist=] or if |json|["lang"] is
           not a [=string=], return.
           </li>
-          <li>If calling <a data-cite=
```

Comment:
Don't we still need this? We still want structurally valid tags, no?

Reply:
I discussed this with @aphillips (see https://github.com/w3c/manifest/pull/1101#discussion_r1631528734). As parsing language tags is quite complicated, our proposed solution is to allow all strings and let the OS determine the appropriate language based on the user's locale. If an invalid language tag is specified, it will simply be ignored and not used.

Reply:
Note that the comment is about _matching_ language tags. _Parsing_ language tags is more straightforward.

It's a good idea to check if the tag is what BCP47 calls "well formed". (BCP47 defined two levels of tag conformance. A tag can be grammatically correct ['well formed'] and it can contain only registered subtags with other restrictions ['valid']. Unfortunately, 402 named the "well-formed-checking" thing `IsStructurallyValidLanguageTag`). In general, few specifications require validity checking of the tags, because that requires access to (a copy of) the IANA Language Subtag Registry, but well-formed checking helps users avoid putting invalid strings in.

The canonicalization of language tags is into locale IDs is probably fine. So perhaps put this text back?

Reply:
yeah, let's put this back. We want the make sure that only structurally valid language tags ever get to the OS. The manifest parser should be filtering out any invalid ones.

Reply:
It's back. I'm leaving this conversation open as a reminder that I need to do the same for the language tag keys in the language map.